src/pykeen/nn/modules.py

```diff
@@ -852,9 +853,10 @@ class DistMultInteraction(FunctionalInteraction[FloatTensor, FloatTensor, FloatT
 
 @parse_docdata
 class DistMAInteraction(FunctionalInteraction[FloatTensor, FloatTensor, FloatTensor]):
-    """A module wrapper for the stateless DistMA interaction function.
+    r"""The stateless DistMA interaction function.
```

Comment:
this doc change isn't quite right, now this class supports both

Reply:
DistMA is a state-less interaction function (meaning that there are no parameters involved except the head/relation/tail representation).

Once we have cleaned this, this should become apparent from the class hierarchy (only state-less derive from `FunctionalInteraction`), and maybe we could also add a better general documentation about interactions.

Reply:
Okay then this one is good to go too